aimm_adapters/heald_labview.py

```diff
@@ -519,6 +590,7 @@ def read(self):
         norm_metadata = {
             "Element": {"symbol": element_name, "edge": edge_symbol},
             "common": {"element": {"symbol": element_name, "edge": edge_symbol}},
+            "Translation": changed_columns,
```

Comment:
Consider mapping new name to old name, as a more convenient lookup.

Reply:
All of the changes were added and tested in the latest commit.

Reply:
Did you flip the direction of the translation mapping? I don't see where, but I could easily be confused.

Reply:
Yes, I went back and check the code. I had the variables in the correct order already. There was no need of flipping them. IT happens inside the _normalize_dataframe()_ method in line 421

https://github.com/AI-multimodal/aimm-adapters/pull/15/commits/592bf4d65f090d67a2aea7a6d9012aebd05270a9#diff-088f3e4588584e5307edab834742d4815b6752eb9e13400ddaa0eb0483de881eL424